website/docs/guides/testing.md

```diff
@@ -365,12 +367,13 @@ describe("a server", () => {
 
 ### Test Scope
 
-As hinted at above, there are two separate scopes at play in your tests, _suite_
-scope, and _test_ scope. The lifetime of the suite scope is the entire test run.
-Any task spawned within it can potentially last across multiple test runs. By
-the same token, the _test_ scope is disposed of after every single test is
-finished. Any tasks spawned within it will be halted immediately after the test
-is finished. For example:
+As hinted at above, there are two separate tasks scopes at play in
+your tests: _suite_ scope, and _test_ scope. The lifetime of the suite
+scope task is the entire test run.  Any task spawned within it can
```

Comment:
`scope` and `task` together

Reply:
Oops, that didn't read well did it. Let me have another go.